Assets/Scripts/SimpleTwistControl.lua

```diff
@@ -0,0 +1,44 @@
+local SimpleTwistControl = {
+    Properties = {}
+}
+
+function SimpleTwistControl:OnActivate()
+	self._gravity = -9.81
+	self._gotMessage = false
+
+    self._rotate = Vector3(0,0,0)
+    self._linear = Vector3(0,0,0)
+    
+    self.tickNotificationBus = TickBus.Connect(self);
+    self.twistNotificationBus = TwistNotificationBus.Connect(self);
+end
+
+function SimpleTwistControl:TwistReceived(linear, angular)
+	self._gotMessage = true
+	self._linear = linear
+	self._rotate = angular
+end
+
+function SimpleTwistControl:OnTick(deltaTime, currentTime)
+	if not self._gotMessage then
+		return
+	end
+
+	RigidBodyRequestBus.Event.SetAngularVelocity(self.entityId, self._rotate)
+	
+	-- Local to world linear velocity
+	local worldTM = TransformBus.Event.GetWorldTM(self.entityId)
+	local linearVelocityTransformed = Transform.TransformVector(worldTM, self._linear)
+	
+	RigidBodyRequestBus.Event.ApplyLinearImpulse(self.entityId, Vector3(0,0,self._gravity))
```

Comment:
Is `ApplyLinearImpulse` necessary here? I have noticed some weird robot pose jumps (or rather very unsmooth motion) during the rotation in place when I used this function. So, when I removed it, the rotation was smoother. Of course, it's possible that the real root cause was somewhere else. So, this is just a question :)

Reply:
We just discussed the topic with @adamdbrw . This was a "hard" workaround for gravitational acceleration. Going to rework this. :+1: